student/src/main/java/tds/student/tdslogger/StudentEventLogger.java

```diff
@@ -0,0 +1,36 @@
+package tds.student.tdslogger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.opentestsystem.delivery.logging.EventLoggerBase;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(value = "request", proxyMode = ScopedProxyMode.TARGET_CLASS)
+public class StudentEventLogger extends EventLoggerBase {
+
+  protected String getApp() {
+    return "student";
+  }
+
+  public enum StudentLogEvent implements LogEvent {
+    DETERMINE_NEXT_QUESTION,
+  }
+
+  public enum StudentEventData implements EventData {
+    STUDENT_ID,
+  }
+
+  public StudentEventLogger() {
```

Comment:
I followed a boilerplate with this from proctor. I asked @mtrupkin the same question. I tried removing it and it fails to start up without it. Is a no-arg constructor required? I noted while testing construction that one instance seems to be created before a request comes in. I can see which one it's calling on that first construction.